packages/api-openfin/src/window.js

```diff
@@ -103,8 +105,6 @@ class Window {
       return this;
     }
 
-    MessageService.subscribe('*', 'test2', (message) => console.log(message));
```

Comment:
I did wonder whether that code was really supposed to be there when I refactored!